--- a/main/RFXNames.h	2015-11-06 17:13:06.689483522 +0100
+++ b/main/RFXNames.h	2015-11-06 17:13:25.059482961 +0100
@@ -152,6 +152,8 @@
 	HTYPE_OpenWebNet,			//70
	HTYPE_RaspberryHTU21D,		//71
	HTYPE_AtagOne,				//72
+	HTYPE_JabloDongle,                      //250
+	HTYPE_TurrisThermo,                     //251
 	HTYPE_END
 };
 
--- a/main/RFXNames.cpp	2015-11-06 11:02:06.573496174 +0100
+++ b/main/RFXNames.cpp	2015-11-03 13:26:48.404474650 +0100
@@ -219,6 +219,8 @@
		{ HTYPE_OpenWebNet, "MyHome OpenWebNet" },
		{ HTYPE_RaspberryHTU21D, "I2C sensor HTU21D(F)/SI702x Humidity+Temp" },
		{ HTYPE_AtagOne, "Atag One Thermostat" },
+		{ HTYPE_JabloDongle, "Jablotron USB Turris Dongle" },
+		{ HTYPE_TurrisThermo, "Turris Router CPU + Board Temperature" },
		{ 0, NULL, NULL }
	};
	return findTableIDSingle1 (Table, hType);
