--- a/cpp/examples/MinOZW/Main.cpp	2014-04-29 07:40:57.000000000 +0200
+++ b/cpp/examples/MinOZW/Main.cpp	2014-10-14 09:56:11.000000000 +0200
@@ -261,7 +261,7 @@ int main( int argc, char* argv[] )
 	// The first argument is the path to the config files (where the manufacturer_specific.xml file is located
 	// The second argument is the path for saved Z-Wave network state and the log file.  If you leave it NULL 
 	// the log file will appear in the program's working directory.
-	Options::Create( "../../../config/", "", "" );
+	Options::Create( "/usr/share/openzwave/", "", "" );
 	Options::Get()->AddOptionInt( "SaveLogLevel", LogLevel_Detail );
 	Options::Get()->AddOptionInt( "QueueLogLevel", LogLevel_Debug );
 	Options::Get()->AddOptionInt( "DumpTrigger", LogLevel_Error );
@@ -286,7 +286,7 @@ int main( int argc, char* argv[] )
 #elif WIN32
         string port = "\\\\.\\COM6";
 #else
-	string port = "/dev/ttyUSB0";
+	string port = "/dev/aeon_zstick_v2";
 #endif
 	if ( argc > 1 )
 	{
