--- a/main/mainworker.cpp	2015-11-06 11:02:06.583496174 +0100
+++ b/main/mainworker.cpp	2015-11-03 13:26:48.411141316 +0100
@@ -74,6 +74,8 @@
 #include "../hardware/Pinger.h"
 #include "../hardware/Nest.h"
 #include "../hardware/Thermosmart.h"
+#include "../hardware/JabloDongle.h"
+#include "../hardware/TurrisThermo.h"
 #include "../hardware/Kodi.h"
 #include "../hardware/Netatmo"
 #include "../hardware/HttpPoller.h"
@@ -854,6 +860,12 @@
		//LAN
 		pHardware = new CurrentCostMeterTCP(ID, Address, Port);
 		break;
+	case HTYPE_JabloDongle:
+		pHardware = new CJabloDongle(ID, SerialPort, 57600);
+		break;
+	case HTYPE_TurrisThermo:
+		pHardware = new CTurrisThermo(ID);
+		break;
 	case HTYPE_DomoticzInternal:
 		pHardware = new DomoticzInternal(ID);
 		break;
