--- a/CMakeLists.txt	2015-11-06 17:52:53.866077338 +0100
+++ b/CMakeLists.txt	2015-11-06 17:57:11.766069467 +0100
@@ -235,6 +235,8 @@
 hardware/1Wire/1WireByKernel.cpp
 hardware/1Wire/1WireCommon.cpp
 hardware/1Wire/1WireForWindows.cpp
+hardware/TurrisThermo.cpp
+hardware/JabloDongle.cpp
 hardware/AccuWeather.cpp
 hardware/AnnaThermostat.cpp
 hardware/ASyncSerial.cpp
@@ -584,6 +584,21 @@
   message(STATUS "==== (Please follow http://wiringpi.com/download-and-install/ if you want to use GPIO.)")
 ENDIF(WiringPi)

+FIND_LIBRARY(SENSORS_LIBRARY NAMES libsensors sensors)
+IF(SENSORS_LIBRARY)
+  MESSAGE(STATUS "libsensors library found at: ${SENSORS_LIBRARY}")
+  target_link_libraries(domoticz ${SENSORS_LIBRARY})
+ELSE(SENSORS_LIBRARY)
+  MESSAGE(FATAL_ERROR "libsensors not found on your system")
+ENDIF(SENSORS_LIBRARY)
+
+FIND_PATH(SENSORS_INCLUDE_DIRS NAMES sensors/sensors.h)
+IF(SENSORS_INCLUDE_DIRS)
+  MESSAGE(STATUS "libsensors includes found at: ${SENSORS_INCLUDE_DIRS}")
+ELSE(SENSORS_INCLUDE_DIRS)
+  MESSAGE(FATAL_ERROR "libsensors includes not found on your system")
+ENDIF(SENSORS_INCLUDE_DIRS)
+
 find_path(TELLDUSCORE_INCLUDE NAMES telldus-core.h)
 if (TELLDUSCORE_INCLUDE)
   message(STATUS "Found telldus-core (telldus-core.h), adding tellstick support")
@@ -717,9 +717,9 @@
   option(USE_STATIC_LIBSTDCXX "Build with static libgcc/libstdc++ libraries" YES)
   IF(USE_STATIC_LIBSTDCXX)
     message(STATUS "Using static libgcc/libstdc++")
-    SET(CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "${CMAKE_SHARED_LIBRARY_LINK_C_FLAGS} -static-libgcc -static-libstdc++")
-    SET(CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS "${CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS} -static-libgcc -static-libstdc++")
-    SET(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -static-libgcc -static-libstdc++")
+    SET(CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "${CMAKE_SHARED_LIBRARY_LINK_C_FLAGS} -static-libgcc -static-libstdc++ -lstdc++")
+    SET(CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS "${CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS} -static-libgcc -static-libstdc++ -lstdc++")
+    SET(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -static-libgcc -static-libstdc++ -lstdc++")
   ELSE(USE_STATIC_LIBSTDCXX)
     message(STATUS "Using dynamic libgcc_s/libstdc++")
   ENDIF(USE_STATIC_LIBSTDCXX)

